projects/kit/src/lib/masks/number/utils/stringify-number-without-exp.ts

```diff
@@ -5,17 +5,44 @@
  * @return string representation of a number
  */
 export function stringifyNumberWithoutExp(value: bigint | number): string {
+    if (typeof value === 'bigint') {
+        return value.toString();
+    }
+
     const valueAsString = String(value);
-    const [numberPart = '', expPart] = valueAsString.split('e-');
+    const [numberPart = '', expPart] = valueAsString.split('e');
+
+    if (!expPart) {
+        return valueAsString;
+    }
+
+    const isNegativeExp = expPart.startsWith('-');
+    const exp = Math.abs(Number(expPart));
+
+    if (!isNegativeExp) {
+        return value.toLocaleString('fullwide', {useGrouping: false});
```

Comment:
Why `fullwide` is required ? Could you provide example when it does not work without it ?

Reply:
fixed